docs/src/man/reshaping_and_pivoting.md

```diff
@@ -61,13 +61,13 @@ d = stackdt(iris)
 
 This saves memory. To create the view, several AbstractVectors are defined:
 
-`:variable` column -- `EachRepeatedVector`  
```

Comment:
Actually, these probably are important. 2 spaces == newline. I forgot this is markdown interpreter dependant